### ISSUE TITLE
Add XCVario Driver

### ISSUE THREAD

**Issue by XCNav** [rdunning0823/tophat#28]
Data and Infos can be found here!
https://github.com/iltis42/XCVario

**Comment by CazYokoyama** [comment]
I am confused.
My glider has XCVario and I connected tophat running on Kobo and Android phone with XCVario through WiFi and Bluetooth.

**Comment by XCNav** [comment]
Yes, you can connect it as XCV supports protocols as borgelt and cambridge. But those support XCV only partially. AHRS for example is not forwarded in any of those protocols, also bucks and ballast are not bidirectional supported

**Comment by CazYokoyama** [comment]
Are you asking something more than http://www.caztech.com/glider/tophat/android/xcvario/?

**Comment by XCNav** [comment]
Thank you!
